lib/api_hammer/sinatra.rb

```diff
@@ -167,5 +167,25 @@ def parsed_body
         halt_error(415, errors)
       end
     end
+
+    # for methods where parameters which are required in the path may also be specified in the 
+    # body, this takes the path_params and the body object and ensures that they are consistent 
+    # any place they are specified in the body.
+    def check_params_and_object_consistent(path_params, object)
+      errors = {}
+      path_params.each do |(k, v)|
+        if object.key?(k) && object[k] != v
```

Comment:
what is `object` here? the body of the http request ?
you are comparing the parameters from the query string and the post request body? am I getting this right?

so if the key is not present in the body, it's alright. if it is, it must match.

I don't know why you name this `object`

also, if I am not mistaken, `|(k, v)|` makes no difference over `|k, v|`